exercise_000_sudoku_solver_initial_state/README.md

```diff
@@ -0,0 +1,65 @@
+# An Akka Typed/Scala based Sudoku Solver - Initial State
+
+## Background
+
+This application implements an Akka Typed/Scala based Sudoku Solver that
+consists of 29 actors.
+
+The application will also start up an instance of a Sudoku problem generator.
+
+We will first show you how to run the application.
+
+
+## Steps
+
+- In the project's root folder, start an `sbt` session
+- Run the Sodukosolver by executing the `runSolver` command from the `sbt` prompt
+- Note that you can stop the application by hitting `Return` in the sbt session
+- Observe sudoku solver in action: you should see the following output:
+
+```scala
+man [e] > Scala 2 to Scala 3 > clustered sudoku solver initial state > runSolver
```

Comment:
man [e] > Scala 2 to Scala 3 > ~~clustered~~ sudoku solver initial state >

Reply:
Good catch!